org.eclipse.scanning.sequencer/src/org/eclipse/scanning/sequencer/analysis/JythonDevice.java

```diff
@@ -0,0 +1,29 @@
+package org.eclipse.scanning.sequencer.analysis;
+
+import org.eclipse.january.dataset.IDataset;
+import org.eclipse.scanning.api.device.models.JythonModel;
+import org.eclipse.scanning.api.scan.ScanningException;
+import org.eclipse.scanning.jython.JythonObjectFactory;
+
+public class JythonDevice extends SlicingRunnableDevice<JythonModel> {
+	
+	private JythonObjectFactory<IJythonFunction> factory;
+
+	@Override
+	public void configure(JythonModel model) throws ScanningException {
+		super.configure(model);
+		this.factory = new JythonObjectFactory<>(IJythonFunction.class, model.getModuleName(), model.getClassName(), "org.eclipse.scanning.sequencer");
+	}
+
+	@Override
+	boolean process(SliceDeviceContext context) throws ScanningException {
+		
+		IJythonFunction jython = factory.createObject();
+		IDataset ret = jython.function(context.getSlice());
+		
+		System.out.println(ret);

```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS106)